posthog/api/test/test_organization_feature_flag.py

```diff
@@ -44,3 +51,190 @@ def test_get_feature_flag_unauthorized(self):
+            "deleted": False,
+            "created_by": self.user.id,
+            "id": "__ignore__",
+            "created_at": "__ignore__",
+            "usage_dashboard": "__ignore__",
+            "is_simple_flag": True,
+            "experiment_set": [],
+            "surveys": [],
+            "features": [],
+            "rollback_conditions": None,
+            "performed_rollback": False,
+            "can_edit": True,
+            "analytics_dashboards": [],
+            "has_enriched_analytics": False,
+            "tags": [],
+        }
+
+        flag_response = response.json()["success"][0]
+
+        for key, expected_value in expected_flag_response.items():
+            self.assertIn(key, flag_response)
+            if expected_value != "__ignore__":
+                if key == "created_by":
+                    self.assertEqual(flag_response[key]["id"], expected_value)
+                else:
+                    self.assertEqual(flag_response[key], expected_value)
+
+        self.assertSetEqual(
+            set(expected_flag_response.keys()),
+            set(flag_response.keys()),
+        )
+
+    def test_copy_feature_flag_update_existing(self):
+        target_project = self.team_2
+        rollout_percentage_existing = 99
+        self.feature_flag_existing = FeatureFlag.objects.create(
+            team=target_project,
+            created_by=self.user,
+            key=self.feature_flag_key,
+            name="Existing flag",
+            filters={"groups": [{"rollout_percentage": rollout_percentage_existing}]},
+            rollout_percentage=rollout_percentage_existing,
+            ensure_experience_continuity=False,
+        )
+
+        url = f"/api/organizations/{self.organization.id}/feature_flags/copy_flags"
+
+        data = {
+            "feature_flag_key": self.feature_flag_to_copy.key,
+            "from_project": self.feature_flag_to_copy.team_id,
+            "target_project_ids": [target_project.id],
+        }
+        response = self.client.post(url, data)
+
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        self.assertIn("success", response.json())
+        self.assertIn("failed", response.json())
+
+        # Check copied flag in the response
+        expected_flag_response = {
+            "key": self.feature_flag_to_copy.key,
+            "name": self.feature_flag_to_copy.name,
+            "filters": self.feature_flag_to_copy.filters,
+            "active": self.feature_flag_to_copy.active,
+            "ensure_experience_continuity": self.feature_flag_to_copy.ensure_experience_continuity,
+            "rollout_percentage": self.rollout_percentage_to_copy,
+            "deleted": False,
+            "created_by": self.user.id,
+            "id": "__ignore__",
+            "created_at": "__ignore__",
+            "usage_dashboard": "__ignore__",
+            "is_simple_flag": True,
+            "experiment_set": [],
+            "surveys": [],
```

Comment:
in a follow up, add survey/experiment/early-access features linked to this flag, and ensure they aren't overridden :)